rtf_vis_tool/src/Components/DivGraph.js

```diff
@@ -0,0 +1,164 @@
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 53)} leftOffset={formatPercent(leftShift, 53)} text={'Bundler Calibrator'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 60)} text={'absolute Ts'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 68)} text={'1d-SfM'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 76)} text={'relative Ts (2): i2ti1'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 86)} text={'Largest Connected Component Extractor'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 63)} leftOffset={formatPercent(leftShift, 65)} text={'absolute Rs'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 65)} leftOffset={formatPercent(leftShift, 76)} text={'Shonan'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 65)} leftOffset={formatPercent(leftShift, 87)} text={'relative Rs (2): i2Ri1'}/>
+        
+                {/* Render Directed Edges */}
+                {arrowList}
+
+
+                {/* Render Plates */}
+                <div className="scene_optimizer_plate">
+                    <p style={{color: 'red', fontWeight: 'bold'}}>Scene Optimizer Scenes</p>
+                </div>
+                <div className="feature_extractor_plate">
+                    <p style={{color: 'red', fontWeight: 'bold'}}>Feature Extractor Images</p>
+                </div>
+                <div className="two_view_estimator_plate" onClick={(fs_json) ? (() => toggleFrontEndSummaryDisplay(true)) : (null)}>
+                    <p style={{color: 'red', fontWeight: 'bold'}}>TwoViewEstimator</p>
+                </div>
+                <div className="averaging_plate">
+                    <p style={{color: 'red', fontWeight: 'bold'}}>Averaging</p>
+                </div>
+                <div className="sparse_multiview_optimizer_plate" onClick={(mvo_json) ? (() => toggleMVOMetrics(true)) : (null)}>
+                    <p style={{color: 'red', fontWeight: 'bold'}}>Sparse Multiview Optimizer</p>
+                </div>
+                <div className="dense_multiview_optimizer_plate">
+                    <p style={{color: 'red', fontWeight: 'bold'}}>Dense Multiview Optimizer</p>
+                </div>
+            </div>
+        </div>
+    )
+}
+
+export default DivGraph;
```

Comment:
DivGraph.js serves as the landing page graph. Renders 43 nodes, the edges connecting them, and the 5 plates.